spikewidgets/widgets/multicompgraphwidget/multicompgraphwidget.py

```diff
@@ -100,8 +100,8 @@ def plot_multicomp_agreement_by_sorter(multi_sorting_comparison, plot_type='pie'
         The colormap to be used for the nodes (default 'Reds')
     figure: matplotlib figure
         The figure to be used. If not given a figure is created
-    ax: matplotlib axis
-        The axis to be used. If not given an axis is created
+    axes: list of matplotlib axis
+        The axese to be used. If not given the required axes are created
```

Comment:
@mhhennig can you add `show_legend` in the docstring?

Reply:
done